.github/workflows/update_types.yml

```diff
@@ -0,0 +1,75 @@
+name: Check Saleor release and update types
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 18 * * 1-5" # workdays at 18:00
+
+jobs:
+  check_saleor_release:
+    name: Check Saleor release and update types
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Get latest release info
+        id: release
+        run: |
+          echo "Fetching latest release info..."
+          latest_release=$(curl -s https://api.github.com/repos/saleor/saleor/releases/latest)
+          echo "tag=$(echo $latest_release | jq -r '.tag_name')" >> "$GITHUB_OUTPUT"
+          echo "has_schema=$(echo $latest_release | jq -e '.assets[] | select(.name == "schema.graphql") | length > 0')" >> "$GITHUB_OUTPUT"
+
+      - name: Download schema.graphql
+        run: |
+          if [[ "${{ steps.release.outputs.has_schema }}" == "true" ]]; then
+            echo "schema.graphql is attached to the ${{ steps.release.outputs.tag }} release, updating..."
+          else
+            echo "schema.graphql is not attached to the ${{ steps.release.outputs.tag }} release, exiting..."
+            exit 1
+          fi
+
+      - uses: robinraju/release-downloader@v1.8
```

Comment:
We could probably not use this action and just use CURL do download it? It's a single file and we already have the path from the previous steps.

Reply:
Done